modules/cudacodec/src/ffmpeg_video_source.cpp

```diff
@@ -131,6 +131,8 @@ cv::cudacodec::detail::FFmpegVideoSource::FFmpegVideoSource(const String& fname)
     format_.codec = FourccToCodec(codec);
     format_.height = cap.get(CAP_PROP_FRAME_HEIGHT);
     format_.width = cap.get(CAP_PROP_FRAME_WIDTH);
+    format_.displayArea = Rect(0, 0, cap.get(CAP_PROP_FRAME_WIDTH), cap.get(CAP_PROP_FRAME_HEIGHT));
```

Comment:
It makes sense to reuse already fetched `.get()` results instead of querying a new one.

Reply:
Agreed :thumbsup: